ush/forecast_postdet.sh

```diff
@@ -0,0 +1,959 @@
+    $NCP $FIX_AM/qr_acr_qs.dat .
+    sleep 60
+  fi
+  $NLN $FIX_AM/${O3FORC}                         $DATA/global_o3prdlos.f77
+  $NLN $FIX_AM/${H2OFORC}                        $DATA/global_h2oprdlos.f77
+  $NLN $FIX_AM/global_solarconstant_noaa_an.txt  $DATA/solarconstant_noaa_an.txt
+  $NLN $FIX_AM/global_sfc_emissivity_idx.txt     $DATA/sfc_emissivity_idx.txt
+  
+  ## merra2 aerosol climo
+  if [ $IAER -eq "1011" ]; then
+    FIX_AER="${FIX_DIR}/fix_aer"
+    for month in $(seq 1 12); do
+      MM=$(printf %02d $month)
+      $NLN "${FIX_AER}/merra2.aerclim.2003-2014.m${MM}.nc" "aeroclim.m${MM}.nc"
+    done
+    FIX_LUT="${FIX_DIR}/fix_lut"
+    $NLN $FIX_LUT/optics_BC.v1_3.dat $DATA/optics_BC.dat
+    $NLN $FIX_LUT/optics_OC.v1_3.dat $DATA/optics_OC.dat
+    $NLN $FIX_LUT/optics_DU.v15_3.dat $DATA/optics_DU.dat
+    $NLN $FIX_LUT/optics_SS.v3_3.dat $DATA/optics_SS.dat
+    $NLN $FIX_LUT/optics_SU.v1_3.dat $DATA/optics_SU.dat
+  fi
+  
+  $NLN $FIX_AM/global_co2historicaldata_glob.txt $DATA/co2historicaldata_glob.txt
+  $NLN $FIX_AM/co2monthlycyc.txt                 $DATA/co2monthlycyc.txt
+  if [ $ICO2 -gt 0 ]; then
+    for file in $(ls $FIX_AM/fix_co2_proj/global_co2historicaldata*) ; do
+      $NLN $file $DATA/$(echo $(basename $file) | sed -e "s/global_//g")
+    done
+  fi
+  
+  $NLN $FIX_AM/global_climaeropac_global.txt     $DATA/aerosol.dat
+  if [ $IAER -gt 0 ] ; then
+    for file in $(ls $FIX_AM/global_volcanic_aerosols*) ; do
+      $NLN $file $DATA/$(echo $(basename $file) | sed -e "s/global_//g")
+    done
+  fi
+  
+  # inline post fix files
+  if [ $WRITE_DOPOST = ".true." ]; then
+    $NLN $PARM_POST/post_tag_gfs${LEVS}             $DATA/itag
+    $NLN $PARM_POST/postxconfig-NT-GFS-TWO.txt      $DATA/postxconfig-NT.txt
+    $NLN $PARM_POST/postxconfig-NT-GFS-F00-TWO.txt  $DATA/postxconfig-NT_FH00.txt
+    $NLN $PARM_POST/params_grib2_tbl_new            $DATA/params_grib2_tbl_new
+  fi
+  
+  #------------------------------------------------------------------
+  # changeable parameters
+  # dycore definitions
+  res=$(echo $CASE |cut -c2-5)
+  resp=$((res+1))
+  npx=$resp
+  npy=$resp
+  npz=$((LEVS-1))
+  io_layout="1,1"
+  #ncols=$(( (${npx}-1)*(${npy}-1)*3/2 ))
+  
+  # spectral truncation and regular grid resolution based on FV3 resolution
+  JCAP_CASE=$((2*res-2))
+  LONB_CASE=$((4*res))
+  LATB_CASE=$((2*res))
+  
+  JCAP=${JCAP:-$JCAP_CASE}
+  LONB=${LONB:-$LONB_CASE}
+  LATB=${LATB:-$LATB_CASE}
+  
+  LONB_IMO=${LONB_IMO:-$LONB_CASE}
+  LATB_JMO=${LATB_JMO:-$LATB_CASE}
+  
+  # Fix files
+  FNGLAC=${FNGLAC:-"$FIX_AM/global_glacier.2x2.grb"}
+  FNMXIC=${FNMXIC:-"$FIX_AM/global_maxice.2x2.grb"}
+  FNTSFC=${FNTSFC:-"$FIX_AM/RTGSST.1982.2012.monthly.clim.grb"}
+  FNSNOC=${FNSNOC:-"$FIX_AM/global_snoclim.1.875.grb"}
+  FNZORC=${FNZORC:-"igbp"}
+  FNAISC=${FNAISC:-"$FIX_AM/CFSR.SEAICE.1982.2012.monthly.clim.grb"}
+  if [ $cplflx = ".false." ] ; then
+    FNALBC2=${FNALBC2:-"$FIX_AM/global_albedo4.1x1.grb"}
```

Comment:
Most of these _probably_ need to be updated to use tiled fix files the same as for coupled runs. Need confirmation from someone more knowledgeable. That can be done later though.

Reply:
Agreed, I think everyone is moving to the tiled fix files not just coupled, however if this change isn't in develop having this if statement is necessary for reproducing the develop branch.